vispy/visuals/mesh.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, vertices=None, faces=None, vertex_colors=None,
         Visual.__init__(self, **kwargs)
         
         self.set_gl_state('translucent', depth_test=True,
-                          cull_face='front_and_back')
+                          cull_face=False)
```

Comment:
At first I thought this would cause a change in behavior, but now I'm thinking that it won't because on `master` just the property type of culling gets set but the culling isn't enabled...?